java/test/src/main/java/test/Ice/stream/Test.ice

```diff
@@ -55,7 +55,7 @@ class OptionalClass
     byte by;
     optional(1) short sh;
     optional(2) int i;
-    optional(3) SmallStruct sm;
+    optional(3) LargeStruct s;
```

Comment:
The renaming from `sm` to `s` is intentional, since `sm` no longer makes sense (I assume it stood for SMallstruct).

This field only exists in the Java mapping, and it's never even set... So there was no where else to change the name.